docs/installation.rst

```diff
@@ -31,7 +31,7 @@ Or add this to your composer.json configuration:
 
 	{
 		"require" : {
-			"FriendsOfCake/crud": "~4.0"
+			"FriendsOfCake/crud": "~4.2"
```

Comment:
I would actually remove this, directly editing the composer.json file is discouraged (Lest we waken the [DOHMSKRAKEN](https://twitter.com/dohmsraken)) - [source](http://blog.doh.ms/2014/10/13/installing-composer-packages/)